src/components/services/AirshipInstance.tsx

```diff
@@ -19,9 +19,20 @@ export interface ShowErrorWarningOptions {
  */
 export function showError(error: unknown, options: ShowErrorWarningOptions = {}): void {
   const { trackError = true, tag } = options
-  const translatedError = tag ? `Tag: ${tag}. ` + translateError(error) : translateError(error)
+  const translatedError = translateError(error)
   if (trackError) {
-    Bugsnag.notify(`showError: ${translatedError}`)
+    const tagMessage = tag == null ? '' : `Tag: ${tag}. `
+    let bugsnagError: Error | string
+    if (error instanceof Error) {
+      bugsnagError = error
```

Comment:
This case should also attempt a translateError and tack on the result to the tagMessage.

Reply:
I tack on the tag to the `message` property of the error below. I don't want to run `translateError` on error objects because it returns a string then we'd lose the stack property of the error.